.github/dependabot.yml

```diff
@@ -9,19 +9,19 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - "Technical Dept"
+      - "Technical Debt"
```

Comment:
Das musst du auch in der .github/release.yml anpassen

Reply:
Done